glide.lock

```diff
@@ -1,21 +1,54 @@
 hash: 7d1e85d070980bf11cab0f4dc869c2af64ecfaf41b1ccb52eaa5db06184b23ba
-updated: 2017-05-03T14:11:01.313181441-07:00
+updated: 2017-06-02T11:28:58.980126559-07:00
```

Comment:
nit: if you're just updating glide.lock, without updating glide.yaml, put that in a separate commit, with an explanation comment as to why, or at least mention it in the check-in comments. I figured it out -- you wanted the new stuff from the idl -- but it took some snooping.